examples/notebooks/python/agaricus-gpu.ipynb

```diff
@@ -201,22 +200,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": 17,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Transformation takes 2.63 seconds\n",
+      "Transformation takes 2.31 seconds\n",
       "+-----+--------------------+--------------------+----------+\n",
       "|label|       rawPrediction|         probability|prediction|\n",
       "+-----+--------------------+--------------------+----------+\n",
-      "|  1.0|[-0.9667757749557...|[0.03322422504425...|       1.0|\n",
-      "|  0.0|[-0.0080436170101...|[0.99195638298988...|       0.0|\n",
-      "|  0.0|[-0.0080436170101...|[0.99195638298988...|       0.0|\n",
-      "|  0.0|[-0.1416745483875...|[0.85832545161247...|       0.0|\n",
-      "|  0.0|[-0.0747678577899...|[0.92523214221000...|       0.0|\n",
+      "|  1.0|[-0.5428417325019...|[0.45715826749801...|       1.0|\n",
+      "|  0.0|[-0.4572062194347...|[0.54279378056526...|       0.0|\n",
+      "|  0.0|[-0.4572062194347...|[0.54279378056526...|       0.0|\n",
+      "|  0.0|[-0.4514296054840...|[0.54857039451599...|       0.0|\n",
+      "|  0.0|[-0.5428417325019...|[0.45715826749801...|       1.0|\n",
```

Comment:
I don't think we need to update this file.
Seems the difference is output with a worse prediction? Maybe the train dataset is too small?

Reply:
restored